monai/networks/nets/__init__.py

```diff
@@ -53,7 +53,25 @@
 from .generator import Generator
 from .highresnet import HighResBlock, HighResNet
 from .hovernet import Hovernet, HoVernet, HoVerNet, HoverNet
-from .mednext import MedNeXt
+from .mednext import (
```

Comment:
@rcremese I would prefer to do a factory method pattern similar to how it exists for resnets: https://github.com/Project-MONAI/MONAI/blob/59a7211070538586369afd4a01eca0a7fe2e742e/monai/networks/nets/resnet.py#L36

for consistency

Reply:
@surajpaib, for sure I can make it look more like resnets factory method.

Reply:
Well, I took exemple on [densnet](https://github.com/Project-MONAI/MONAI/blob/19cc6f01766120132f964beecb06d1d561f83801/monai/networks/nets/densenet.py#L152) and [SEnet](https://github.com/Project-MONAI/MONAI/blob/19cc6f01766120132f964beecb06d1d561f83801/monai/networks/nets/senet.py#L50) which are both integrated into the core library.

Reply:
I see the inconsistency now. 

Factory methods are better in this case I would say - it keeps them simpler as we are not using any functionality provided by subclassing the MedNext parent. Moreover, if we want to add custom logic like loading some pre-trained weights, these lend better too.